Makefile

```diff
@@ -1,6 +1,7 @@
-.PHONY: clean install analysis test test-install develop docs docs-install
+.PHONY: clean venv install analysis test test-install develop docs docs-install
```

Comment:
I think we can assume that if there is a venv directory then we've previously installed and so python must be installed, dont think we need to rerun that check every time.